flink-libraries/flink-table/pom.xml

```diff
@@ -252,6 +261,29 @@ under the License.
 									<pattern>org.eigenbase</pattern>
 									<shadedPattern>org.apache.flink.shaded.calcite.org.eigenbase</shadedPattern>
 								</relocation>
+								<relocation>
+									<pattern>org.pentaho</pattern>
+									<shadedPattern>org.apache.flink.shaded.calcite.org.pentaho</shadedPattern>
+								</relocation>
+
+								<!-- flink-table dependencies -->
+								<relocation>
+									<pattern>org.apache.commons</pattern>
+									<shadedPattern>org.apache.flink.shaded.org.apache.commons</shadedPattern>
```

Comment:
For flink dependencies, please change the shading pattern to `org.apache.flink.table.shaded` to a) make them more unique and b) distinguish them from flink-shaded dependencies.